source/broadband/webpa_notification.c

```diff
@@ -354,6 +378,114 @@ void *FactoryResetCloudSync()
 	return NULL;
 }
 
+void SyncNotifyRetryTask()
+{
+	int err = 0;
+	pthread_t threadId;
+	err = pthread_create(&threadId, NULL, SyncNotifyRetry, NULL);
+	if (err != 0)
+	{
+		WalError("Error creating SyncNotifyRetry thread :[%s]\n", strerror(err));
+	}
+	else
+	{
+		WalInfo("SyncNotifyRetry Thread created Successfully\n");
+		g_syncRetryThreadStarted = 1;
+		g_checkSyncNotifyRetry = 1;
+	}
+}
+
```

Comment:
Add a comment here, 

//When CPE and cloud are out of sync, retry sync notification twice with a 7mins delay between attempts